lib/windows/winlean.nim

```diff
@@ -437,7 +437,7 @@ type
     fd_count*: cint # unsigned
     fd_array*: array[0..FD_SETSIZE-1, SocketHandle]
 
-  AddrInfo* = object
+  AddrInfo* {.importc: "ADDRINFOA", header: "ws2tcpip.h".} = object
```

Comment:
ASCII structure in 2025, you joking?

Reply:
[ADDRINFOA](https://learn.microsoft.com/en-us/windows/win32/api/ws2def/ns-ws2def-addrinfoa) is applicable to [getaddrinfo](https://learn.microsoft.com/en-us/windows/win32/api/ws2tcpip/nf-ws2tcpip-getaddrinfo). If you want [ADDRINFOW](https://learn.microsoft.com/en-us/windows/win32/api/ws2def/ns-ws2def-addrinfow), then you should use [GetAddrInfoW](https://learn.microsoft.com/en-us/windows/win32/api/ws2tcpip/nf-ws2tcpip-getaddrinfow). 

Currently, the Nim standard library uses getaddrinfo, and the GetAddrInfoW function is not imported. Users may need to import it themselves.

https://github.com/nim-lang/Nim/blob/10c9ebad9303d9c4be393da913f4e12650783539/lib/windows/winlean.nim#L571-L576